client/src/modules/reports/generate/employee_standing/employee_standing.js

```diff
@@ -12,21 +12,28 @@ EmployeeStandingController.$inject = [
  * @description
 
  */
-function EmployeeStandingController($state, $sce, Notify, BaseReportService, AppCache, SavedReports, reportData) {
+function EmployeeStandingController($state, $sce, Notify, AppCache, SavedReports, reportData, Session) {
 
   const vm = this;
   const cache = new AppCache('configure_employee_standing');
-  const reportUrl = '/reports/finance/employee_standing';
 
   vm.reportDetails = {};
 
   checkCachedConfiguration();
 
+  vm.reportDetails.currency_id = Session.enterprise.currency_id;
```

Comment:
Note:  Doing it this way forces the form to use the enterprise currency each time you enter the page.  The cache is ignored.  Look at the checkCachedConfiguration() function in client/src/modules/reports/generate/avg_med_costs_per_patient/avg_med_costs_per_patient.js for an example.

Reply:
Nice